addons/sms/wizard/sms_composer.py

```diff
@@ -290,6 +290,13 @@ def _prepare_body_values(self, records):
         return all_bodies
 
     def _prepare_mass_sms_values(self, records):
+        """ Prepare sms values for the given records.
+
+        :returns {tuple(recordset, dict)}:
```

Comment:
no need for {} I think, makes it seem like we might return a set

But also we already return a dict of res_id -> values.
Can't we just use the keys of the dict to know which ones have a value?